README.md

```diff
@@ -75,6 +75,7 @@ This library currently supports the following cryptocurrencies and address forma
  - SYS (base58check P2PKH and P2SH, and bech32 segwit)
  - TRX (base58check)
  - VET (checksummed-hex)
+ - VLX (base58)
```

Comment:
Please change this to base58, no check (and revolve conflict again)

Reply:
Done.